lib/puppet_litmus/rake_tasks.rb

```diff
@@ -100,7 +100,7 @@ def run_local_command(command)
     config_data = { 'modulepath' => File.join(Dir.pwd, 'spec', 'fixtures', 'modules') }
     raise "the provision module was not found in #{config_data['modulepath']}, please amend the .fixtures.yml file" unless File.directory?(File.join(config_data['modulepath'], 'provision'))
 
-    if %w[abs vmpooler].include?(args[:provisioner])
+    if %w[abs vmpooler docker vagrant].include?(args[:provisioner])
```

Comment:
Perhaps it should be an early raise? Then dedent the rest of the block. I'm also wondering if supported provisioners should be a constant to avoid the duplication.

```suggestion
    unless %w[abs vmpooler docker vagrant].include?(args[:provisioner])
      raise "Unknown provisioner '#{args[:provisioner]}', try abs/docker/vmpooler/vagrant"
    end
```